de.akaflieg_freiburg.cavok.add_hours_and_minutes.json

```diff
@@ -0,0 +1,28 @@
+{
+    "app-id": "de.akaflieg_freiburg.cavok.add_hours_and_minutes",
```

Comment:
Do you own this domain exist? If not, `com.gitlab.kebekus.addHoursAndMinutes` would be a better choice.